content/en/tutorials/real-time-multiplayer-colyseus.md

```diff
@@ -0,0 +1,403 @@
+
+> *Select create game to open a new game. And click anywhere on the floor to move the object.*
+
+## On this tutorial you will learn:
+- Setting up your Colyseus server
+- Synchronizing the state between server and client
+- Exchanging messages between client and server
+- Matchmaking: how to create, join, and list available games
+
+## Materials
+
+- [PlayCanvas Project (Client-side)](https://playcanvas.com/project/859259/overview/colyseus-demo)
+- [Colyseus TypeScript Project (Server-side)](https://github.com/lpsandaruwan/colyseus-playcanvas-server/) ([Glitch Server](https://glitch.com/edit/#!/tutorial-playcanvas-server))
+
+# Before you start
+
+## Prior Knowledge Expected
+
+- Basic PlayCanvas knowledge ([See PlayCanvas Developer Resources](https://developer.playcanvas.com/))
+- Basic JavaScript/TypeScript understanding ([See TypeScript Handbook](https://www.typescriptlang.org/docs/handbook/intro.html))
+- Basic Node.js understanding ([See Introduction to Node.js](https://nodejs.dev/learn/))
+
+## Software requirements
+
+- [Node.js LTS](https://nodejs.org/en/download/)
+- [Visual Studio Code](https://code.visualstudio.com/download)
+
+# Creating the Server
+
+We will be making a basic server for keeping player states. Changes will be synchronized with clients accordingly.
+
+To create a fresh new Colyseus server, run the following from your command-line:
+
+```
+npm init colyseus-app ./playcanvas-demo-server
+```
+
+Let's make sure you can run the server locally now, by running `npm start`:
+
+```
+cd playcanvas-demo-server
+npm start
+```
+
+If successful, the output should look like this in your command-line:
+
+```
+> my-app@1.0.0 start
+> ts-node-dev --respawn --transpile-only src/index.ts
+
+✅ development.env loaded.
+✅ Express initialized
+🏟 Your Colyseus App
+⚔️ Listening on ws://localhost:2567
+```
+
+# Including the Colyseus JavaScript SDK
+
+Now we need to add the Colyseus JavaScript SDK on PlayCanvas.
+
+We can do that through a "external script" on PlayCanvas project settings.
+
+Open **"Menu" → "Settings"**:
+
+![settings](/images/tutorials/multiplayer-colyseus/settings.png)
+
+From the Settings panel, expand on **"External Scripts"**, and increase the number of **"URLs"**.
+
+![CDN](/images/tutorials/multiplayer-colyseus/ext_script.png)
+
+In the new **"URL"** field, let's include the Colyseus JavaScript SDK, from a CDN:
+
+```
+https://unpkg.com/colyseus.js@^0.14.0/dist/colyseus.js
+```
+
+This is going to make the `Colyseus` [JavaScript SDK](https://docs.colyseus.io/colyseus/getting-started/javascript-client/) available for our PlayCanvas scripts.
+
+# Establishing a Client-Server Connection
+
+Now, from a new PlayCanvas Script, let's instantiate our `Colyseus.Client` instance. ([see "Creating new scripts"](https://developer.playcanvas.com/en/user-manual/scripting/creating-new/))
+
+You can attach this script to any object in your scene, such as the Camera.
```

Comment:
I think we can help diffuse some project best practices and instead suggest to add this script to a new empty entity (named something like "NetworkManager").